.github/workflows/test-all-benchmarks-schedule.yml

```diff
@@ -0,0 +1,92 @@
+name: Test all benchmarks on schedule
+
+on:
+  push:
+#  TODO: Remove
+    branches:
+      - MD-1043-evgeny-benchmarks
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 6"  # Runs every Saturday at 12 AM UTC
+
+jobs:
+  test-all-benchmarks:
+    runs-on: self-hosted
+    timeout-minutes: 720
```

Comment:
12 hours not enough, try to increase to 18hours

Reply:
This is still 12 hours, we could double it to 24h

Reply:
It is an outdated comment.
We should be good if we run it in parallel.